environments.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Environments": [
+    {
+      "id": "login",
+      "url": "https://login.salesforce.com/",
+      "user name": "",
+      "user password": ""
```

Comment:
When we implement the framework, we will use one single username and password as credentials to login to the application.